src/Parser/MarksmanshipHunter/Modules/Features/CastEfficiency.js

```diff
@@ -94,34 +97,39 @@ class CastEfficiency extends CoreCastEfficiency {
       getCooldown: haste => 24,
       noSuggestion: true,
       noCanBeImproved: true,
+      hideWithZeroCasts: true,
```

Comment:
This should only be used for spells that can't be detected if a player has access to them, like racials. Anything else you should be able to toggle with `isActive`.

Reply:
isActive implies that they don't have access to them if set to false, despite them having access to it. I would think this would only show them on fights where binding shot, concussive shot etc are useful, as those fights are somewhat rare. Otherwise it simply clutters the cast efficiency screen.

Reply:
These two images show the massive clutter caused by all these utility spells, and then the image post "fix". If there is some other way to remove them, if they weren't cast at all, then I'll gladly use that - but this was the only one I could find initially. 

<img width="1062" alt="screen shot 2017-10-02 at 21 39 23" src="https://user-images.githubusercontent.com/29204244/31095770-432a5858-a7ba-11e7-86a7-4db29d8ed980.png">
<img width="1077" alt="screen shot 2017-10-02 at 21 39 30" src="https://user-images.githubusercontent.com/29204244/31095769-4328a9a4-a7ba-11e7-9e19-daf8e0a22578.png">

Reply:
I recommend putting them under a utility category if a cluttered view worries you. The intend is for the Cast Efficiency tab to be a complete list of spells to show players everything that's available to them and maybe reveal something they could use that they weren't using yet. Some people forget they even had a certain ability after not using it in a very a long time.

Reply:
I'll move some of them to Defensive instead, I guess, that'll aid in this. 
In other news, wouldn't it make sense to have item above Utility, simply because Items usually are a integral part of dmg/healing, whereas utility may not be? It feels bad to have them relegated to bottom spot.

Reply:
Should be as easy as reordering the items in Parser/Core/Modules/CastEfficiency - SPELL_CATEGORIES